srv/components/system/network/mgmt/bonded/init.sls

```diff
@@ -15,7 +15,7 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-{%- if (pillar['cluster'][node]['network']['mgmt_nw']['ipaddr']) and (not pillar['cluster'][grains['id']]['network']['mgmt_nw']['gateway']) %}
+{%- if (pillar['cluster'][node]['network']['mgmt'ipaddr']) and (not pillar['cluster'][grains['id']]['network']['mgmt']['gateway']) %}
```

Comment:
This needs to be corrected:
**['mgmt']['ipaddr']**

Reply:
fixed.
Should be: pillar['cluster'][node]['network']['mgmt']['public_ip']